<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitCanDialog</class>
 <widget class="QDialog" name="InitCanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>361</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CAN通道初始化</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-144</y>
        <width>270</width>
        <height>450</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>450</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labProtocol">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>协议</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboProtocol"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labCanfdStandard">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>CANFD标准</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboCanfdStandard"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labCanfdSpeedUp">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>CANFD加速</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboCanfdSpeedUp"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labAbit">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>仲裁域波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboAbit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="labDbit">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>数据率波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboDbit">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labWorkMode">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>工作模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboWorkMode">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="labResistance">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>终端电阻</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboResistance">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>上报总线利用率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_8">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>总线利用周期(ms)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_9">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>发送重试</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_10">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
